runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -682,12 +684,470 @@ J9::ARM64::TreeEvaluator::asynccheckEvaluator(TR::Node *node, TR::CodeGenerator
 TR::Register *
 J9::ARM64::TreeEvaluator::instanceofEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   // Call helper
-   TR::ILOpCodes opCode = node->getOpCodeValue();
-   TR::Node::recreate(node, TR::icall);
-   TR::Register *targetRegister = directCallEvaluator(node, cg);
-   TR::Node::recreate(node, opCode);
-   return targetRegister;
+   TR::Compilation *comp = cg->comp();
+
+   if (comp->getOption(TR_DisableInlineInstanceOf))
+      {
+      // Call helper
+      TR::ILOpCodes opCode = node->getOpCodeValue();
+      TR::Node::recreate(node, TR::icall);
+      TR::Register *targetRegister = directCallEvaluator(node, cg);
+      TR::Node::recreate(node, opCode);
+      return targetRegister;
+      }
+   else
+      {
+      return VMinstanceofEvaluator(node, cg);
+      }
+   }
+
+/**
+ * @brief Generates instructions to load j9class from object pointer
+ * 
+ * @param[in]       node: node
+ * @param[in] j9classReg: register j9class value is assigned to
+ * @param[in]     objReg: register holding object pointer
+ * @param[in]         cg: code generator
+ */
+static void 
+generateLoadJ9Class(TR::Node* node, TR::Register* j9classReg, TR::Register *objReg, TR::CodeGenerator* cg)
```

Comment:
Two styles are mixed here for variable declaration:
- `TR::Register* j9classReg` (space after `*`)
- `TR::Register *objReg` (space before `*`)

Reply:
Changed the style of all declarations to the latter.